boxoffice/static/js/views/admin_discount_policy.js

```diff
@@ -0,0 +1,512 @@
+                  return {
+                    results: data.items
+                  };
+                },
+                cache: true
+              },
+              escapeMarkup: function (markup) { return markup; },
+              templateResult: function(item) {
+                let markup = '<p>' + item.title + '</p>';
+                return markup;
+              },
+              templateSelection: function(item) {
+                return item.title;
+              }
+            });
+
+            $(startDateSelector).daterangepicker({
+              singleDatePicker: true,
+              showDropdowns: true,
+              timePicker: true,
+              timePickerSeconds: true,
+              timePicker24Hour: true,
+              opens: 'left',
+              locale: {
+                format: 'D MMM YYYY H:mm:ss'
+              }
+            });
+
+            $(endDateSelector).daterangepicker({
+              singleDatePicker: true,
+              showDropdowns: true,
+              timePicker: true,
+              timePickerSeconds: true,
+              timePicker24Hour: true,
+              opens: 'left',
+              locale: {
+                format: 'D MMM YYYY H:mm:ss'
+              }
+            });
+          }
+          else {
+            let addItemsSelector;
+
+            if(discountPolicy) {
+              let dpId = discountPolicyComponent.get(discountPolicy + '.id');
+              addItemsSelector = "#add-items-" + dpId;
+            }
+            else {
+              addItemsSelector = "#add-items";
+            }
+
+            $(addItemsSelector).select2({
+              minimumInputLength: 3,
+              multiple: true,
+              placeholder: 'Search tickets',
+              ajax: {
+                url: OrgModel.urlFor('view_items', {org_name: discountPolicyComponent.get('org')})['path'],
+                dataType: 'json',
+                data: function (params) {
+                  return {
+                    search: params.term
+                  };
+                },
+                processResults: function (data) {
+                  console.log("processResults", data.items);
+                  return {
+                    results: data.items
+                  };
+                },
+                cache: true
+              },
+              escapeMarkup: function (markup) { return markup; },
+              templateResult: function(item) {
+                let markup = '<p>' + item.title + '</p>';
+                return markup;
+              },
+              templateSelection: function(item) {
+                return item.title;
+              }
+            });
+          }
+        },
+        showNewPolicyForm: function(event) {
+          discountPolicyComponent.set('showAddPolicyForm', true);
+          discountPolicyComponent.set('newDiscountPolicy.is_price_based', 1);
+          discountPolicyComponent.set('newDiscountPolicy.discount_type', 1);
```

Comment:
Perhaps better to store these values (eg: 1) as constants, document them and use those constants, instead of hard-coding the values here.